packages/page/src/Application/MessageHandler/MovePageMessageHandler.php

```diff
@@ -24,11 +29,43 @@ class MovePageMessageHandler
 {
     public function __construct(
         private PageRepositoryInterface $pageRepository,
+        private DomainEventCollectorInterface $domainEventCollector,
+        private RequestAnalyzerInterface $requestAnalyzer,
     ) {
     }
 
     public function __invoke(MovePageMessage $message): void
     {
+        $page = $this->pageRepository->getOneBy($message->getIdentifier());
+        $previousParent = $page->getParent();
+
         $this->pageRepository->moveOneBy($message->getIdentifier(), $message->getTargetParentIdentifier());
+
+        /** @var ?string $locale */
+        $locale = $this->requestAnalyzer->getAttribute('locale');
```

Comment:
Add the missing locale to the message and remove the dependency to the requestanalyzer